tests/data/ingress-class/no-class-ingress.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  annotations:
```

Comment:
I think you can remove `annotations` if empty?

Reply:
I was thinking to keep it as a hint: "look, annotations are empty". I can remove this for sure

Reply:
Ah ok, I am ok with that too :)